### PR TITLE
SHARE-21 removed nolb

### PR DESCRIPTION
**Pull request by dmetzner**
removed everything connected to nolb

### CODE REVIEW

File: src/Catrobat/Controller/Ci/BuildApkController.php [hunk 1]
```diff
@@ -23,13 +23,12 @@ class BuildApkController extends Controller
    * @Route("/ci/build/{id}", name="ci_build", defaults={"_format": "json"},
    *   requirements={"id": "\d+"}, methods={"GET"})
    *
-   * @param Request $request
    * @param Program $program
    *
    * @return JsonResponse
    * @throws \Exception
    */
-  public function createApkAction(Request $request, Program $program)
```
Comment (HCrane) [inline src/Catrobat/Controller/Ci/BuildApkController.php:32]: Why did you remove that?

File: tests/behat/features/api/register_and_login.feature [hunk 2]
```diff
@@ -60,7 +60,7 @@ Feature: Login with an existing account or register a new one
     When I POST these parameters to "/pocketcode/api/login/Login.json"
     Then I should get the json object:
       """
-      {"statusCode":200, "token": "rrrrrrrrrrr", "email":"default1@pocketcode.org","nolbUser":false,"preHeaderMessages":""}
+      {"statusCode":200, "token": "rrrrrrrrrrr", "email":"default1@pocketcode.org","preHeaderMessages":""}
```
Comment (HCrane) [inline tests/behat/features/api/register_and_login.feature:63]: Could be formated a little bit more beautiful but oh well :)

**Review by HCrane — APPROVED** [review]

File: src/Catrobat/Controller/Ci/BuildApkController.php (hunk 1 above)
Comment (dmetzner) [reply src/Catrobat/Controller/Ci/BuildApkController.php:32]: 'cause it was unused... 
(just produces a warning)
